docs/container.md

```diff
@@ -62,23 +62,26 @@ run through the Xailient SDK.
 Replace `<OUTPUT_IMAGES_DIR>` with the __full path__ to a directory where the images and videos with their predicted bounding
 boxes drawn will be placed.
 
-Note: You can also place the downloaded Xailient SDK (a python .whl file) into the "<INPUT_IMAGES_DIR>" directory of the host OS.
- Then you can start the docker container without specifying the Xailient SDK's URL via `-e SDK_LINK="<LINK>"`
+Note: You can also place the downloaded Xailient SDK (a python .whl file) into the `<INPUT_IMAGES_DIR> ` directory of the host OS.
+ Then you can start the docker container without specifying the Xailient SDK's URL via `-e SDK_LINK="<LINK>"` e.g.
 
     `$ docker run -v <FULL_PATH>:/input -v <FULL_PATH>:/output xailient/model-inference:1.0.1`
 
 Providing an `SDK_LINK` takes precedence over placing the Xailient SDK file in the "input" directory. Therefore, be aware when swapping in different Xailient SDKs.
 
 **Example:**
 
-`$ docker run -e SDK_LINK="https://ReallyLongUrl/AAaDDY2MzE1ODQ0NzQx" -v ~/Desktop/input:/input -v ~/Workspace/output:/output xailient/model-inference:1.0.1`
+    `$ docker run -e SDK_LINK="https://ReallyLongUrl/AAaDDY2MzE1ODQ0NzQx" -v ~/Desktop/input:/input -v ~/Workspace/output:/output xailient/model-inference:1.0.1`
 
-All predicted images and videos will appear in the `~/Workspace/output` of the host OS.
+In this example, all predicted images and videos will appear in the `~/Workspace/output` of the host OS.
 
 <br>
 ## Default Behaviour
 
-By default the Xailient SDK container will output images for all given input images, and output videos for all given input videos.
+By default the Xailient SDK container will output images for all given input images, and output videos and their frames for all given input videos.
```

Comment:
My original section of example images and videos disappeared so didn't think it was worth actually giving the user example images anymore.

Reply:
Hmm, did the default for videos really change to output both a video and each frame like this edit suggests?